orchagent/portsorch.cpp

```diff
@@ -3130,6 +3130,37 @@ void PortsOrch::doTask(Consumer &consumer)
     }
 }
 
+bool PortsOrch::flushFDBEntries(sai_object_id_t bridge_port_id)
+{
+    vector<sai_attribute_t>    attrs;
+    sai_attribute_t            attr;
+    sai_status_t               rv = SAI_STATUS_SUCCESS;
+
+    SWSS_LOG_ENTER();
+
+    if (SAI_NULL_OBJECT_ID == bridge_port_id)
+    {
+        SWSS_LOG_WARN("Couldn't flush FDB. Bridge port OID: 0x%" PRIx64,
+                      bridge_port_id);
+        return false;
+    }
+
+    attr.id = SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID;
+    attr.value.oid = bridge_port_id;
+    attrs.push_back(attr);
+
+    SWSS_LOG_INFO("Flushing FDB bridge_port_oid: 0x%" PRIx64, bridge_port_id);
```

Comment:
what are the values of bridge_port_id? a magic number on the sw know of the user also know it?
should we provide a more clear info to the user?

Reply:
yes, now port.m_alias.c_str() is getting printed instead.